src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -313,11 +313,11 @@ private function buildXml(\DOMElement $parentNode, $data, $xmlRootNodeName = nul
                 } elseif ($key === '#') {
                     $append = $this->selectNodeType($parentNode, $data);
                 } elseif (is_array($data) && false === is_numeric($key)) {
-                    /**
+                    /*
                      * Is this array fully numeric keys?
```

Comment:
I think this comment is obsolete as it just tells _what_ the line below does, not _why_ it does it. Maybe it should be removed instead?